### PR TITLE
Improving types for polkadot's api webview implementation

### PR DESCRIPTION
**Pull request by silva-fj**
This is the initial PR to add implement polkadot's api through the webview.

Despite our setup works fine, it is quite easy to make mistakes but not so easy to debug when breaking something in the polkadot-api module or in the app side, this initial PR aims to reduce these kind of issues by improving our types.

The integration of the API will come in a separate PR.

Let's re-test the wallet functionalities to be sure we are safe.

### CODE REVIEW

File: src/context/TxContext.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ import {MessageTeaser} from '@ui/components/MessageTeaser';
 import {Subheading, Caption, Icon, useBottomSheet, Button} from '@ui/library';
 import globalStyles, {standardPadding} from '@ui/styles';
 import {Padder} from '@ui/components/Padder';
-import {SignCredentials} from '@polkadotApi/types';
+import {SignCredentials} from 'polkadot-api';
```
Comment (ishrak-imam) [inline src/context/TxContext.tsx]: NIT 💅 
```suggestion
import type {SignCredentials} from 'polkadot-api';
```

File: src/polkadotApi/atoms.ts [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
-  AddAccountPayload,
-  AddExternalAccountPayload,
-  RestoreAccountPayload,
-  ExportAccountPayload,
-  Accounts,
-  SignCredentials,
-  SignedMessage,
-} from './types';
 import {persistAtom} from '@atoms/persist';
+import {
```
Comment (ishrak-imam) [inline src/polkadotApi/atoms.ts]: NIT 💅 
```suggestion
import type {
```

File: src/ui/components/Tx/AuthenticateView.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import {Padder} from '@ui/components/Padder';
 import {SecureKeychain} from 'src/service/SecureKeychain';
 import globalStyles, {standardPadding} from '@ui/styles';
 import {useKeyring} from '@polkadotApi/useKeyring';
-import {SignCredentials} from '@polkadotApi/types';
+import {SignCredentials} from 'polkadot-api';
```
Comment (ishrak-imam) [inline src/ui/components/Tx/AuthenticateView.tsx]: NIT 💅 
```suggestion
import type {SignCredentials} from 'polkadot-api';
```

**Review by ishrak-imam — APPROVED** [review]
The type improvements looks really good.

**Review by Ekaanth — APPROVED** [review]

**Pull request merged**